src/app/organizations/state/register-organization.service.ts

```diff
@@ -138,7 +138,7 @@ export class RegisterOrganizationService {
         location: organizationFormState.location,
         email: organizationFormState.contactEmail,
         // Setting approved to true to appease compiler.  Webservice should completely ignore this.
```

Comment:
Even if the value is ignored, why not set it to something more appropriate here like 
Organisation.StatusEnum.PENDING?
(and below)